modules/foundation/handlers.go

```
@@ -0,0 +1,205 @@
+	})
+}
+
+// healthHandler handles health check requests
+func healthHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	response := map[string]interface{}{
+		"status":  "healthy",
+		"service": "user-service",
+		"version": "1.0.0",
+	}
+	json.NewEncoder(w).Encode(response)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)